main-site/components/sponsor-us-section/whySponsor/WhySponsor.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import {
+  StyledTextContainer,
+  StyledParagraph,
+  StyledWhySponsorContainer,
+  StyledCenterImage,
+  StyledItemContainer,
+  StyledFennecFox,
+  StyledHeaderLabel
+} from './WhySponsor.styles';
+
+import { P, H3, H4 } from '../../../../shared-ui/style/typography';
+import { colors } from '../../../../shared-ui/style/colors';
+import FennecFox from '../../../../shared-ui/images/fennec-fox.svg';
+import { Col, Row } from 'antd';
```

Comment:
Great job implementing this grid library! Next time you want to use a new technology run it by me or the tech channel!